app/controllers/dashboard/advisers_controller.rb

```diff
@@ -8,8 +9,31 @@ def index
     end
 
     def new
-      @firm = Firm.find(params[:firm_id])
-      @adviser = advisers.build
+      @adviser = @firm.advisers.build
+    end
+
+    def create
+      @adviser = @firm.advisers.build(adviser_params)
+      @adviser.confirmed_disclaimer = true
+      # ^^^ This needs to be resolved by some heavier refactoring
+      # work across mas-rad_core and this codebase.
```

Comment:
Note to other reviewers: there is a Trello card already on our board for the work needed here.